compiler/src/dmd/clone.d

```diff
@@ -1609,68 +1671,38 @@ private Statement generateCopyCtorBody(StructDeclaration sd)
  *
  * Params:
  *  sd = the `struct` for which the copy constructor is generated
- *  hasCpCtor = set to true if a copy constructor is already present
+ *  hasCopyCtor = set to true if a copy constructor is already present
  *  hasMoveCtor = set to true if a move constructor is already present
  *
  * Returns:
  *  `true` if one needs to be generated
  *  `false` otherwise
  */
```

Comment:
`src/dmd/clone.d(1681): Warning: Ddoc: parameter count mismatch, expected 5, got 3`

Reply:
yes, I'll have it fixed shortly